go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/paketo-buildpacks/pipeline-builder-stool-pigeon
+
+go 1.20
+
+replace github.com/buildpacks/libcnb => github.com/buildpacks/libcnb v1.24.1-0.20230627130937-5d3d3f6e6d16
+
+replace github.com/paketo-buildpacks/libpak => github.com/paketo-buildpacks/libpak v1.66.1-0.20230707152440-cd6875914a49
```

Comment:
It could be some of the weirdness with octo but you don't normally need these replace items to target a commit. You can just `go get` the specific commit, and it'll add it to the require block, like on lines 10 and 12.

Reply:
I'll try this.. I think it's probably a left over from where I was using replace to point at my repos during testing.. if it works I'll revisit the ones in pipeline-builder too..